app/views/content_items/manuals/_manual_layout.html.erb

```diff
@@ -1,8 +1,10 @@
 <% content_for :main do %>
-  <%= render "govuk_publishing_components/components/heading", {
-    text: t("manuals.summary_title"),
-    margin_bottom: 4
-  } %>
+  <% if @content_item.description.present? %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("manuals.summary_title"),
+      margin_bottom: 4
+    } %>
+  <% end %>
```

Comment:
Not a blocking comment but: one thing that seems odd about this section is that the summary title and the text of the summary are separated in such a way that they need two separate if statements testing whether the description is present. It feels a bit like if the description is inside the body, the summary title ought to be too. But probably an FE dev would have to make that call. As things stand, this code change is fine.

Reply:
Oh yeah good point. Let me do that and see if it looks OK and get a FE dev to 👀